frame/privilege/src/lib.rs

```diff
@@ -148,7 +161,70 @@ pub mod pallet {
 	// FIXME: Temporary fix to get CI to pass, separate PRs will be made per pallet to refactor to
 	// use OptionQuery instead
 	#[allow(clippy::disallowed_types)]
-	pub type GroupCount<T: Config> = StorageValue<_, u32, ValueQuery>;
+	pub type GroupCount<T: Config> = StorageValue<
+		_,
+		u32,
+		ValueQuery,
+		Counter<ZeroInit, IncrementToMax<T::MaxGroup, TooManyGroup, Error<T>>, SafeDecrement>,
+	>;
+
+	error_to_pallet_error!(
+		TooManyGroup -> TooManyGroup;
+	);
+
+	// #[derive(::core::fmt::Debug, ::core::default::Default, ::core::cmp::PartialEq)]
+	// pub struct TooManyGroup;
+
+	// impl<T: Config> From<TooManyGroup> for Error<T> {
+	// 	fn from(_: TooManyGroup) -> Error<T> {
+	// 		Error::<T>::TooManyGroup
+	// 	}
+	// }
+
+	// pub trait IntoDispatchError<TError, T>
+	// where
+	// 	Self: Into<TError>,
+	// 	TError: Into<DispatchError>,
+	// {
+	// 	fn into_dispatch_error(self) -> DispatchError;
+	// }
+
+	// impl<TError, T> IntoDispatchError<TError, T> for T
```

Comment:
why it is commented?

Reply:
tried to do something that didn't work, forgot to remove it. removed now :+1: